drivers/include/pca9633.h

```diff
@@ -0,0 +1,336 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_pca9633 PCA9633 I2C PWM controller
+ * @ingroup     drivers_actuators
+ * @brief       Device driver for the NXP PCA9633
+ *
+ * @{
+ *
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ * @file
+ */
+
+#ifndef PCA9633_H
+#define PCA9633_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "stdbool.h"
+#include "periph/i2c.h"
+
+#include "pca9633_regs.h"
+
+/**
+ * @brief Blinking period with a maximum duration of ~10.73 s
```

Comment:
nitpick: 10625 ms is actually ~10.63 s ;)

Reply:
`10.73 s` is actually  the value I got from the data sheet. Let me check.

Reply:
```
A fifth 8-bit resolution (256 steps) Group PWM controller
has both a fixed frequency of 190 Hz and an adjustable frequency between 24 Hz to once
every 10.73 seconds with a duty cycle that is adjustable from 0 % to 99.6 % that is used to
either dim or blink all LEDs with the same value.
```

Reply:
The 10.73 appears several times so I would conclude it's not a type there. Probably that's just the value with inaccuracies in mind.